Assignment/solution_student/resume.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+    <title>Joe Johnson | Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<header>
+
+<h2><em><a href="index.html">About Me</a></em> | Portfolio</h2>
+<h1>Joe Johnson</h1>
+
+</header>
+
+<body>
+
+<hr>
+
+<h2>Experience</h2>
+        
+<p>I recently graduated from a Front-End Web Development course at General Assembly where I learned HTML, CSS, JavaScript, jQuery and how to be an <b>awesome</b> Front-End Web Developer! During my spare time, I enjoy senior water aerobics, pickling, and spending time with my goldfish, Jerry.</p>
+
+<h3>Portfolio</h3>
+	<ul>
```

Comment:
You have each item in the experience list marked up in a separate `<ul>` element, but single-item lists kinda defeat the purpose of the _list_, which should list _more than one thing_. So what I'd do instead is markup all of each item as nested content in an `<li>`, for example:
```
<ul>
	<li>
		<h3>Web Developer, Relaxr <em>2015 - present</em></h3>
		<p>Developed a multi-column layout blog, landing page, and contact forms that render on mobile devices.</p>
		<img src="images/relaxr_deliverable.png" alt="Relaxr">
	</li>
	<li>
		<h3>Web Developer, Startup Matchmaker <em>2015 - present</em></h3>
		<p>Developed a multi-column layout blog, landing page, and contact forms that render on mobile devices.</p>
		<img src="images/startup_matchmaker_deliverable.png" alt="Startup Matchmaker">
	</li>
	<!-- etc. -->
</ul>
```